Code/Framework/AzFramework/AzFramework/Input/System/InputSystemComponent.cpp

```diff
@@ -162,6 +162,18 @@ namespace AzFramework
         incompatible.push_back(AZ_CRC("InputSystemService", 0x5438d51a));
     }
 
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    void InputSystemComponent::GetDependentServices([[maybe_unused]] AZ::ComponentDescriptor::DependencyArrayType& dependent)
+    {
+        dependent.push_back(AZ_CRC("NativeUIInputSystemService", 0x67675d29));
```

Comment:
This should use AZ_CRC_CE
```suggestion
        dependent.push_back(AZ_CRC_CE("NativeUIInputSystemService"));
```